dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -93,21 +92,24 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
     }
     return message.toString();
   }
-
+  
   /**
-   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
-   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
-   * values are all {@link ClassPathEntry}s in the subtree of the key.
+   * Returns the classpath entries whose transitive dependencies contain
+   * the artifact with the specified coordinates.
```

Comment:
It's the other way around. The artifact with the specified coordinates contains the class path entries as (transitive and direct) dependencies.

Reply:
fixed